cmd/pipecd/ops.go

```diff
@@ -79,6 +80,35 @@ func (s *ops) run(ctx context.Context, t cli.Telemetry) error {
 		return err
 	}
 
+	// Prepare sql database.
+	if cfg.Datastore.Type == model.DataStoreMySQL {
```

Comment:
nits: This is quite similar to the Firestore's one starting at line 158, hence why not making them close together?

Reply:
It sounds good 👍 Just to confirm, is it okay to move `the Firestore's one` as close to this new defined session (the MySQL's one)? The MySQL SQL prepare session required to be defined before `createDatastore` session so that it must be there as it is 👀

Reply:
Let's do it! There's o particular reason to put Firestore's one 👍

Reply:
🙆‍♀️

Reply:
addressed on https://github.com/pipe-cd/pipe/pull/1699/commits/4564c51717c1098db27ecdd2330fd3dd06e4eb34 🙏